deployments/nfd/node-feature-rules.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureRule
+metadata:
+  name: intel-dp-devices
+spec:
+  rules:
+    - name: "intel.dlb"
+      labels:
+        "intel.feature.node.kubernetes.io/dlb": "true"
+      matchFeatures:
+        - feature: pci.device
+          matchExpressions:
+            vendor: {op: In, value: ["8086"]}
+            device: {op: In, value: ["2710"]}
+            class: {op: In, value: ["0b40"]}
+        - feature: kernel.loadedmodule
+          matchExpressions:
+            dlb2: {op: Exists}
+
+    - name: "intel.dsa"
+      labels:
+        "intel.feature.node.kubernetes.io/dsa": "true"
+      matchFeatures:
+        - feature: pci.device
+          matchExpressions:
+            vendor: {op: In, value: ["8086"]}
+            device: {op: In, value: ["0b25"]}
+            class: {op: In, value: ["0880"]}
+        - feature: kernel.loadedmodule
+          matchExpressions:
+            idxd: {op: Exists}
+
+    - name: "intel.fpga-arria10"
+      labels:
+        "intel.feature.node.kubernetes.io/fpga-arria10": "true"
+      matchFeatures:
+        - feature: pci.device
+          matchExpressions:
+            vendor: {op: In, value: ["8086"]}
+            device: {op: In, value: ["09c4"]}
+            class: {op: In, value: ["1200"]}
+        - feature: kernel.loadedmodule
+          matchExpressions:
+            dfl_pci: {op: Exists}
+
+    - name: "intel.gpu"
+      labels:
+        "intel.feature.node.kubernetes.io/gpu": "true"
+      matchFeatures:
+        - feature: pci.device
+          matchExpressions:
+            vendor: {op: In, value: ["8086"]}
+            class: {op: In, value: ["0300"]}
+        - feature: kernel.loadedmodule
+          matchExpressions:
+            drm: {op: Exists}
```

Comment:
@bart0sh @uniemimu would `i915` be better here?

Reply:
I didn't see it running on our GPU testing host.

Reply:
@eero-t you tweaked these for us, could you align with Mikko to something that works best?